projects/packages/my-jetpack/_inc/components/my-jetpack-screen/styles.module.scss

```
@@ -35,9 +47,49 @@
 	& :global(.components-notice__dismiss) {
 		align-self: center;
 	}
+
+	button, button:visited, button:active, button:hover {
+		color: var( --jp-white );
+		font-weight: 600;
+		font-size: 16px;
+		line-height: 24px;
+		letter-spacing: -0.01em;
+		text-decoration: none;
+		cursor: pointer;
+		justify-content: center;
+		align-items: center;
+		padding: 8px 24px;
+		margin-top: 24px;
+		background: #000000;
+		border-radius: 4px;
+		height: auto;
+
+		@include break-medium {
+			margin-top: 0;
+			white-space: nowrap;
+		}
+	}
 }
 
 .message {
-	margin-left: 8px;
-}
+	margin-right: 10px;
```

```suggestion
        margin-right: var( --spacing-base ); // 8px
```

We should not move out of `8px` rule.